docs/update_release_notes.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Script to help with the SMQTK release process. Performs the following steps:
+#   - Poetry version (major, minor, or patch)
+#   - Rename release_notes/pending_release file to release_notes/version
+#   - Add reference to new release notes file in release_notes.rst
+#   - Add new release notes stub file
+#
+# Two git commits are created. One for the version bump and one for the new
+# release notes stub file.
+#
+
+# Check args
+if [ $# != 1 ]
+then
+  echo "Please enter valid version bump type. Options: major, minor, or patch"
+  exit 1
+fi
+
+if [ $1 != 'major' ] && [ $1 != 'minor' ] && [ $1 != 'patch' ]
+then
+  echo "Please enter valid version bump type. Options: major, minor, or patch"
+  exit 1
+fi
+
+RELEASE_NOTES_PATH="release_notes"
+RELEASE_TYPE=$1
+
+# Update version
+poetry version $1
+
+# Get version
+VERSION=$(poetry version -s)
+
+echo "Current version: ${VERSION}"
+
+# Rename release notes to current version
+cp ${RELEASE_NOTES_PATH}/pending_release.rst ${RELEASE_NOTES_PATH}/v${VERSION}.rst
+
+echo "Created v${VERSION}.rst"
+
+# Add reference to new file in release_notes.rst
+echo "   release_notes/v${VERSION}" >> release_notes.rst
+
+echo "Reference added to release_notes.rst"
+
+# Make new pending notes file
+echo "Pending Release Notes" > ${RELEASE_NOTES_PATH}/pending_release.rst
+echo "=====================\n" >> ${RELEASE_NOTES_PATH}/pending_release.rst
+echo "Updates / New Features" >> ${RELEASE_NOTES_PATH}/pending_release.rst
+echo "----------------------\n" >> ${RELEASE_NOTES_PATH}/pending_release.rst
+echo "Fixes" >> ${RELEASE_NOTES_PATH}/pending_release.rst
+echo "-----\n" >> ${RELEASE_NOTES_PATH}/pending_release.rst
```

Comment:
Maybe a more "elegant" thing to do here is to add a `.pending_notes_stub.rst` file (or some other filename) that we just copy instead of needing this sequence of echos.